docs/content/Getting-Started/Getting-Started-Docker-Compose.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Docker Compose
+permalink: /getting-started/docker/compose
+category: Getting Started
+menuOrder: 3
+---
+
+This guide will help you get Cube.js running using Docker Compose.
+
+> LOOM VIDEO HERE
+
+## 1. Create a Docker Compose file
+
+Create a `docker-compose.yml` file with the following content:
+
+```yaml
+version: '2.2'
+
+services:
+  cube:
+    image: cubejs/cube:latest
+    ports:
+      # 4000 is a port for Cube.js API
+      - 4000:4000
+      # 3000 is a port for Playground web server
+      # it is available only in dev mode
+      - 3000:3000
+    environment:
+      - CUBEJS_DEV_MODE=true
+    volumes:
+      - cube:/cube/conf
+
+volumes:
```

Comment:
@hassankhan I'm not very familiar with docker compose. Is it necessary?

Reply:
Not necessary at all; in fact doing it this way means files aren't on the host filesystem, they're within a Docker volume (similar to the Docker one-liner command). It would be more convenient to mount a local folder and remove this section altogether.